system-tests/__snapshots__/reporters_spec.js

```diff
@@ -624,11 +661,11 @@ Because this error occurred during a \`after all\` hook we are skipping the rema
 
   (Screenshots)
 
-  -  /XXX/XXX/XXX/cypress/screenshots/simple_failing_hook.cy.js/simple failing hook s     (1280x720)
+  -  /XXX/XXX/XXX/cypress/screenshots/simple_failing_hook.cy.js/simple failing hook s    (2560x1440)
      pec -- beforeEach hooks -- never gets here -- before each hook (failed).png                    
-  -  /XXX/XXX/XXX/cypress/screenshots/simple_failing_hook.cy.js/simple failing hook s     (1280x720)
+  -  /XXX/XXX/XXX/cypress/screenshots/simple_failing_hook.cy.js/simple failing hook s    (2560x1440)
      pec -- afterEach hooks -- runs this -- after each hook (failed).png                            
-  -  /XXX/XXX/XXX/cypress/screenshots/simple_failing_hook.cy.js/simple failing hook s     (1280x720)
+  -  /XXX/XXX/XXX/cypress/screenshots/simple_failing_hook.cy.js/simple failing hook s    (2560x1440)
```

Comment:
Why would this change increase the snapshot size? This seems quite large for a default snapshot size.

Reply:
that's the size of the launched browser